contracts/KeyManager.sol

```diff
@@ -156,4 +154,43 @@ contract KeyManager is Ownable {
     return keysByPurpose[_purpose];
   }
 
+  /**
+   * @dev Convert an Ethereum address (20 bytes) to an ERC725 key (32 bytes)
+   * by keccak256-ing it in order to avoid byte shifting and preserving privacy
+   * @param addr address 20 bytes eth address
+   */
+  function addressToKey(address addr)
```

Comment:
Why not remove this method completely? 
We could manage the delegate or executeRightAddresses directly.
Something like:
```javascript
//store the delegates
mapping (address => boolean) delegates;

// set delegate to true 
addDelegate(address delegate) onlyOwner, onlyDelegate { ...}
//modifier
 modifier onlyDelegate() {
        require(delegate[msg.sender] == true);
        _;
    }
```
I think the current pattern is a bit weird. In Ethereum the hash of a public key is the address. 
We take the address hash it again and call the method addressToKey.

At least I think we should change the method name to something like `addressToKeyFormat`.
Because the real ethereum addressToKey is not possible because hashing is a one-way function.
It might be a bit confusing otherwise.

Reply:
What is the main argument against my proposal? Storing the address in an own mapping instead of the keys?

It is not real privacy because the passed address will be always stored inside the block.

Reply:
that is only for the owner. When you add a key you send the keccak version

Reply:
we can also keccak that to have full privacy. The other thing is that you have to shift bytes in case of address. Considering that keys can be something else then eth address how would know what to byte shift in the contract. This way the one who added the key knows if it;s an eth address